executor/Cargo.toml

```diff
@@ -25,6 +25,7 @@ lcs = { path = "../common/lcs", version = "0.1.0", package = "libra-canonical-se
 libra-config = { path = "../config", version = "0.1.0" }
 libra-crypto = { path = "../crypto/crypto", version = "0.1.0" }
 libra-logger = { path = "../common/logger", version = "0.1.0" }
+libra-mempool = { path = "../mempool", version = "0.1.0" }
```

Comment:
nit: why this is required in executor. If it's just for testing, add it to test dependencies only

Reply:
we need it for `CommittedTransaction` defined in shared_mempool. If you think importing mempool here just for this struct is too much, I can move it to libra-types. Lmk